backend/ttrs/models.py

```diff
@@ -61,6 +61,8 @@ def __str__(self):
 
 class TimeTable(models.Model):
     owner = models.ForeignKey('ttrs.Student', related_name='time_tables', on_delete=models.CASCADE)
+    sender = models.ForeignKey('ttrs.Student', related_name='sent_time_tables', on_delete=models.CASCADE, blank=True, null=True)
```

Comment:
I think `on_delete` option should not be `CASCADE`.
Even if the sender leave 'ttrs', the owner must not lose the time table.